dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/RumEventSerializerTest.kt

```diff
@@ -250,7 +251,31 @@ internal class RumEventSerializerTest {
     }
 
     @Test
-    fun `keep known custom attributes as is`(
+    fun `𝕄 serialize RUM event 𝕎 serialize() with unknown event`(
+        @Forgery fakeEvent: RumEvent,
+        @Forgery unknownEvent: UserInfo
+    ) {
+        val rumEvent = fakeEvent.copy(event = unknownEvent)
+
+        val serialized = underTest.serialize(rumEvent)
+
+        val jsonObject = JsonParser.parseString(serialized).asJsonObject
+        assertEventMatches(jsonObject, rumEvent)
+        assertThat(jsonObject)
+            .doesNotHaveField("type")
+            .doesNotHaveField("date")
+            .doesNotHaveField("error")
+            .doesNotHaveField("action")
+            .doesNotHaveField("resource")
+            .doesNotHaveField("application")
+            .doesNotHaveField("session")
+            .doesNotHaveField("view")
+            .doesNotHaveField("usr")
+            .doesNotHaveField("_dd")
```

Comment:
This or we could make `Any.toJson(): JsonElement` throw an exception on unrecognised type, to not malform the payload (IIRC, what I've noticed early on iOS is that  RUM Intake rejects payload if it contains inconsistent entry).